client/me/billing-history/main.jsx

```diff
@@ -18,63 +17,40 @@ var observe = require( 'lib/mixins/data-observe' ),
 	PurchasesHeader = require( '../purchases/list/header' ),
 	BillingHistoryTable = require( './billing-history-table' ),
 	UpcomingChargesTable = require( './upcoming-charges-table' ),
-	SectionHeader = require( 'components/section-header' ),
-	puchasesPaths = require( 'me/purchases/paths' );
+	SectionHeader = require( 'components/section-header' );
+
+import Main from 'components/main';
+import puchasesPaths from 'me/purchases/paths';
 
 module.exports = React.createClass( {
 	displayName: 'BillingHistory',
 
 	mixins: [ observe( 'billingData', 'sites' ), eventRecorder ],
 
-	componentWillMount: function() {
-		classes( document.body ).add( 'billing-history-page' );
-	},
-
-	componentWillUnmount: function() {
-		classes( document.body ).remove( 'billing-history-page' );
-	},
-
 	render: function() {
 		var data = this.props.billingData.get();
 		const hasBillingHistory = ! isEmpty( data.billingHistory );
 
 		return (
-
-			<div id="billing-history-wrapper" className="main">
+			<Main className="billing-history">
 				<MeSidebarNavigation />
 				<PurchasesHeader section={ 'billing' } />
-				<SectionHeader label={ this.translate( 'Receipts' ) } />
-				<Card id="billing-history-content">
-					<div className="billing-history-header">
-						<p>
-							{
-								this.translate(
-									'A complete history of all billing transactions for your WordPress.com account. If you are looking to add or cancel a plan go to {{link}}Manage Purchases{{/link}}.', {
-										components: {
-											link: <a href={ puchasesPaths.list() }
-												onClick={ this.recordClickEvent( 'Manage Purchases Link on Billing History' ) } />
-										}
-									}
-								)
-							}
-						</p>
-					</div>
-
+				<Card className="billing-history__receipts">
 					<BillingHistoryTable transactions={ data.billingHistory } />
 				</Card>
-
+				<Card href={ puchasesPaths.list() }>
+					{ this.translate( 'Go to "Purchases" to add or cancel a plan.' ) }
```

Comment:
Could you explain why we added this section? Is that because users miss the link in the header cake?